src/Symfony/Component/Console/Tests/Command/ListCommandTest.php

```diff
@@ -80,8 +80,7 @@ public function testExecuteListsCommandsOrder()
   -h, --help            Display help for the given command. When no command is given display help for the list command
   -q, --quiet           Do not output any message
   -V, --version         Display this application version
-      --ansi            Force ANSI output
-      --no-ansi         Disable ANSI output
+      --[no-]ansi       Force ANSI output
```

Comment:
Before the no option has "Disable ANSI...." now you can understand it as "do not force it, but do not disable it"

Shall we add add some magic for the description? Or shall we add sth to provide a description for the negatable option?

Reply:
You're right. What's about?
* `Force/Disable ANSI output`
* `Force [or Disable] ANSI output`

Reply:
I like the second one 👍🏻